projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3337,10 +3338,11 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         });
 
         this.verticalScrollContainer.contentSizeChange.pipe(filter(() => !this._init), destructor).subscribe(() => {
-            this.notifyChanges();
+            this.notifyChanges(true);
```

Comment:
This might regress this issue: https://github.com/IgniteUI/igniteui-angular/issues/11476
It should be re-tested.

Reply:
Doesn't exactly regress it, but the scrollbar jumps around in a strange way after this change:
![jumping](https://user-images.githubusercontent.com/10397980/175069106-e09bd71e-022f-41c4-b83b-86b2799d9ade.gif)

Reply:
This behavior is not introduced from my change ... i have tested it to a clear 14.0.x branch and this behave in the same way
![scroll14 0 x](https://user-images.githubusercontent.com/34240583/175237486-21262c2e-967b-4f3a-a4b0-1e4133c8e598.gif)